src/highdicom/sr/content.py

```diff
@@ -750,9 +750,30 @@ def __init__(
         Parameters
         ----------
         graphic_type: Union[highdicom.sr.GraphicTypeValues3D, str]
-            name of the graphic type
-        graphic_data: Sequence[Sequence[int]]
-            ordered set of (row, column, frame) coordinate pairs
+            name of the graphic type. Permissible values are "ELLIPSOID",
+            "POINT", "ELLIPSE" or "POLYGON".
+        graphic_data: Union[np.ndarray, Sequence[np.ndarray]]
+            List of graphic data for elements of the volume surface. Each item
+            of the list should be a 2D numpy array representing the graphic
+            data for a single element of type ``graphic_type``.
+
+            If ``graphic_type`` is "ELLIPSOID" or "POINT", the volume surface
+            will consist of a single element that defines the entire surface.
+            Therefore, a single 2D numpy array should be passed for
+            ``graphic_data`` as a list of length 1. For reasons of backwards
+            compatibility, it is also possible to pass a numpy array directly.
+
+            If ``graphic_type`` is "ELLIPSE" or "POLYGON", the volume surface
+            will consist of two or more planar regions that together define the
+            surface. Therefore a list of two or more 2D numpy arrays should be
+            passed for ``graphic_data``.
+
+            Each 2D numpy array should have dimension N x 3 where each row of
+            the array represents a coordinate in the 3D frame of reference. The
+            number, N, and meaning of the coordinates depends upon the value of
+            "graphic_type". See :class:`highdicom.sr.GraphicTypeValues3D` for
+            details.
```

Comment:
Minor changes for consistency with other docstrings

```suggestion
            If `graphic_type` is ``"ELLIPSOID"`` or ``"POINT"``, the volume
            surface will consist of a single element that defines the entire
            surface. Therefore, a single 2D NumPy array should be passed
            as a list of length 1. For reasons of backwards compatibility,
            it is also possible to pass a NumPy array directly.

            If `graphic_type` is ``"ELLIPSE"`` or ``"POLYGON"``, the volume
            surface will consist of two or more planar regions that together
            define the surface. Therefore a list of two or more 2D NumPy
            arrays should be passed.

            Each 2D NumPy array should have dimension N x 3 where each row of
            the array represents a coordinate in the 3D Frame of Reference. The
            number, N, and meaning of the coordinates depends upon the value of
            `graphic_type`. See :class:`highdicom.sr.GraphicTypeValues3D` for
            details.
```